website/docs/debuggertype.md

```diff
@@ -10,6 +10,7 @@ debuggertype "value"
 * `Mixed` - needs documentation.
 * `NativeOnly` - needs documentation.
 * `ManagedOnly` - needs documentation.
+* `NativeWithManagedCore` - needs documentation.
```

Comment:
Would be fine to fix docu...

Reply:
Documentation has been updated for all of the supported values